### PR TITLE
[bitnami/postgresql] Added labels for volumeClaimTemplates

### PR DESCRIPTION
**Pull request by rahulsadanandan**
**Description of the change**

Added labels for volumeClaimTemplates

**Benefits**

If we want to delete a pvc using a label release. 
for eg:
kubectl delete pvc -l release=xxx

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/postgresql/templates/statefulset.yaml [hunk 1]
```diff
@@ -562,6 +562,11 @@ spec:
   volumeClaimTemplates:
     - metadata:
         name: data
+        labels:
+          app: {{ template "postgresql.name" . }}
+          release: {{ .Release.Name }}
+          heritage: {{ .Release.Service }}
+          component: master
```
Comment (dani8art) [inline bitnami/postgresql/templates/statefulset.yaml:569]: We should use labels convention

```suggestion
        labels: {{- include "common.labels.standard" . | nindent 10 }}
          app.kubernetes.io/component: master    
```

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @rahulsadanandan thank you for the PR.

Could you take a look at my comment?

**Comment by rahulsadanandan** [comment]
Updated the pr

**Comment by marcosbc** [comment]
Hi, unfortunately we've had to revert this PR. It causes chart upgrades to fail due to changing an immutable field. There is also other stuff like we should be using matchLabels, and not setting `common.labels.standard`. Feel free to re-open a new PR with a new approach to these changes.

**Pull request merged**